### DISCUSSION TITLE
How to join the onbind event of listview?

### DISCUSSION THREAD

**Discussion by Majestychen** [Sketchware-Pro/Sketchware-Pro#1028] [Q&A] [answered]
How to join the onbind event of listview? 
There are no onbind events in the listview of the latest version of sketchware that I installed!

**Comment by j-px** [comment, marked as answer]
to be able to use onBindCustomView, you need to set a custom view in the ListView/RecyclerView

**Comment by Mersi-LLC** [comment]
Is not working when click custompage click is crashed...
